src/test/e2e/api/admin/tags.e2e.test.ts

```diff
@@ -209,3 +209,12 @@ test('Can bulk remove tags', async () => {
         })
         .expect(200);
 });
+
+describe('backward compatibility', () => {
```

Comment:
I'd follow one test convention in a file. If we have test(...) multiple times I wouldn't do describe with a test inside.

Reply:
Sure, I'm happy to unwrap it. It's just the one for now, so it's not a big deal. But I think using describe to group tests related to a single area is useful, and I don't necessarily think that having both root-levels test and describe blocks is bad 🤷🏼